src/Rule/Boolean.php

```diff
@@ -78,20 +80,21 @@ protected function validateValue($value, DataSetInterface $dataSet = null): Resu
         return $result;
     }
 
-    public function getOptions(): array
+    public function getOptions(?ErrorMessageFormatterInterface $formatter = null): array
```

Comment:
I suggest to you to remove that parameter and find and translate ErrorMessage outside of that method.